packages/frontend/src/components/ProjectAccess/ProjectAccessCreation/index.tsx

```diff
@@ -227,17 +231,29 @@ const ProjectAccessCreation: FC<ProjectAccessCreationProps> = ({
                         }}
                     />
 
-                    <SubmitButton
-                        intent={Intent.PRIMARY}
-                        text="Give access"
+                    <Button
+                        variant="filled"
+                        px="xs"
                         type="submit"
+                        radius="xs"
+                        sx={{
+                            marginTop: '20px',
+                            marginLeft: '7px',
+                            backgroundColor: '#2D72D2',
+                            height: '30px',
+                            width: '96px',
+                            fontFamily: 'sans-serif',
+                            fontWeight: 'lighter',
+                        }}
```

Comment:
same here (regarding custom styling). We should try to keep this as _untouched_ as possible, so for example:
* if you want it with less padding, perhaps use `compact`? 
* no custom fonts, background color, margins
* also, try to use the Mantine props for box-model-related styles: `m`/`mx`/`h`/`w`, etc